### PR TITLE
feat: create new env variable resource_group_name

### PR DESCRIPTION
**Pull request by akocbek**
### Description
Issue: https://github.com/terraform-ibm-modules/terraform-ibm-rag-sample-da/issues/70
Setting: https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/scripts/build_setup.sh#L103

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by brendankellyibm — APPROVED** [review]

File: solutions/banking/main.tf [hunk 1]
```diff
@@ -245,6 +245,17 @@ resource "ibm_cd_tekton_pipeline_property" "watsonx_assistant_id_pipeline_proper
   value       = var.watson_assistant_instance_id
 }
 
+# Update CD pipeline with Resource Group
+resource "ibm_cd_tekton_pipeline_property" "resource_group_name_pipeline_property_cd" {
+  depends_on  = [local.cd_instance]
+  count       = var.resource_group_name != null ? 1 : 0
+  provider    = ibm.ibm_resources
+  name        = "dev-resource-group"
+  pipeline_id = var.ci_pipeline_id
```
Comment (in-1911) [inline solutions/banking/main.tf:254]: @akocbek @brendankellyibm @ocofaigh - if this is for CD pipeline - why is it going into var.ci_pipeline_id ?

**Pull request merged**